posts/2019-10-1-serverless-azure-functions-v1-part2.md

```diff
@@ -0,0 +1,337 @@
+---
+title: "How to Create a REST API with Azure Functions and the Serverless Framework - Part 2"
+description: "Learn how to add API Management, Webpack and CI/CD to your Azure Functions REST API"
+date: 2019-10-1
+thumbnail: "https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/azure-functions-part1/thumbnail.png"
+heroImage: "https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/azure-functions-part1/header.png"
+category:
+  - guides-and-tutorials
+authors:
+  - TannerBarlow
+---
+
+#### Overview
+
+Now that you've created and deployed a basic API from [Part 1](https://serverless.com/blog/serverless-azure-functions-v1), let's take a few more steps towards making that API more resilient and secure. This post will still be based on the [example repo](https://github.com/tbarlow12/sls-az-func-rest-api), and will follow the same "commit-per-step" format as [Part 1](https://serverless.com/blog/serverless-azure-functions-v1), which contains Steps 1 and 2.
```

Comment:
Can we give the reader the commands for how to catch up step by step? First step: 
`git clone reponame && git checkout hash-of-step1`. 
For step 2: `git checkout hash-of-step2` 
etc.